src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`